docs/examples/ex13.py

```diff
@@ -63,7 +63,7 @@ def port_flux(w):
 current = {}
 for port, boundary in mesh.boundaries.items():
     fbasis = FacetBasis(mesh, elements, facets=boundary)
-    current[port] = asm(port_flux, fbasis, u=fbasis.interpolate(u))
+    current[port] = asm(port_flux, fbasis, u=u)
```

Comment:
This demonstrates that you don't need to call `Basis.interpolate` anymore if `u` is based on the same `Basis` object.

Reply:
Oh, is this new?  Very nice.  This will make user-level scripts simpler and tidier.  So there will be a few other examples to follow ex13 in this simplification; I'll manage that as an exercise in the new idiom.  Ta.